#id of the Dump that finished
int32 id
#status (success/failure) of the Dump
int32 status